src/components/UserFeedbackComponent.jsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Container, Header, Table, Pagination } from 'semantic-ui-react';
+import feedbackAction from '../_actions/feedback.action';
+import SideMenuComponent from '../_components/SideMenuComponent';
+import TableRowComponent from './TableRowComponent';
+
+export class UserFeedbackComponent extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      activePage: 1,
+      limit: 10,
+    };
+  }
+
+  componentDidMount() {
+    this.props.feedbackAction(this.state.defaultPage);
+  }
+
+  handlePaginationChange = (e, { activePage }) => {
+    this.setState({ activePage });
+    this.props.feedbackAction(activePage);
+  }
+
+  handlePageTotal = () => Math.ceil(this.props.feedbackCount / this.state.limit)
+
+  emptyFeedbackCount = () => (this.props.feedbackCount === 0)
```

Comment:
There is a few issues with this method 

1. `emptyFeedbackCount` sounds like you are counting emptyFeedback which seems like a domain term. A domain term is a word a business or organization as chosen to mean something so everyone can understand what they are trying to communicate

2. Is `this.props.feedbackCount` in sync with `this.props.feedback.lenght` ?

3. Change the method name to something better. Let me know what name you come up with